app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
```

Comment:
You see, in python _ means that it is unimportant variable and will never be used.
```suggestion
    for i in range(len(argv) - 1):
```
If you do so, you don't need var `index`, since it has exactly the same values as i and it is autoincremented. Plus, it's more readable since you don't use `_`